packages/server/src/distributed/redis/subservice.ts

```diff
@@ -101,7 +121,29 @@ export class RedisSubservice implements DistributedSubservice {
     await this.pub.publish(scopedChannel, JSON.stringify({ nodeId: this.nodeId, ...message }))
   }
 
+  async lock(ressource: string) {
+    const ttl = 2000
+    const lock = {
+      lock: await this.redlock.acquire(ressource, ttl),
+      ressource,
+      expiry: new Date(new Date().getTime() + ttl)
+    }
+    this.locks[ressource] = lock
+    return lock
+  }
+
+  async release(lock: RedisLock) {
```

Comment:
What happens if you never call release after acquiring a lock?

Reply:
The lock will time out after 2 seconds. The expiry is stored with the locks, so other nodes can know a lock is expired without having to receive a message.